__main__.py

```
@@ -0,0 +1,600 @@
+
+    # Parse arguments
+    args = parser.parse_args()
+
+    if not args.mode:
+        # Default to exploit mode if target provided
+        if len(sys.argv) > 1 and not sys.argv[1].startswith('-'):
+            args.mode = 'exploit'
+            args.target = sys.argv[1]
+            # Re-parse with defaults
+            remaining_args = sys.argv[2:] if len(sys.argv) > 2 else []
+            args = exploit_parser.parse_args([sys.argv[1]] + remaining_args)
```

Ruff (0.12.2)

Consider `[sys.argv[1], *remaining_args]` instead of concatenation

Replace with `[sys.argv[1], *remaining_args]`

(RUF005)